NEWS.rst

```diff
@@ -5,15 +5,16 @@ New features
 ------------
 
 * #22849: Implemented support for signed SAML AuthN Requests.
-
+* #22381: Replace 'Enhedsnummer' with the a desription the OUs position in the tree
```

Comment:
Bunch of typos.
`#22381: Replace 'Enhedsnummer' with the description the OUs position in the tree`

Reply:
I'd suggest `with a description of the location of the organisational unit`